CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# CHANGELOG
+
+This file documents all notable changes in the`LogDNA Node.js logger package`. The release numbering uses [semantic versioning](http://semver.org).
```

Comment:
could incorporate the information you have in the pr body in the changelog

Reply:
It was my understanding that happens in another PR where we update changelog, bump package.json version, then add git tags.  If it wasn't for circleci doing auto-publishing, I'd use our auto versioner on it so we wouldn't have to do this stuff manually.  @smusali ?

Reply:
@darinspivey, we can add `Unreleased` section containing the changes not released yet but overall, it doesn't matter that much because we need to update `CHANGELOG` when we update the `version` inside `package.json` for releasing new version

Reply:
Yeah, let's just wait.